flow-typed/Claim.js

```diff
@@ -66,6 +66,7 @@ declare type CollectionId = ClaimId | string; // uuid
 
```

Comment:
:small_orange_diamond: Channel indicator immediately after publishing is incorrect

1. Create private list
2. Publish the list under a channel. `collection_create` looks correct (`channel_id` populated), but 
    - :x: The avatar that appears in the page immediately after this action is showing the active channel avatar instead.
        - <img width="200" alt="image" src="https://user-images.githubusercontent.com/64950861/191190465-b1078a7f-4185-45b4-8006-4915e4c48008.png">
    - :x: Leave it to confirm -- it will became "Anonymous" instead.
        - <img width="200" alt="image" src="https://user-images.githubusercontent.com/64950861/191191097-c9d9a455-3d10-4adb-923c-923cb8a74de6.png">


But after an F5, it is showing the correct channel.

Reply:
does this still happen? haven't seen it once, on publish I always get the right channel

Reply:
The first bullet no longer occurs, but if you wait long enough for it to confirm, it still jumps to Anonymous for me.

I did make an Anonymous publish prior to that -- not sure if that matters.